src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            Lottery lottery = new Lottery();
+            Ball newBall = Lottery.getRandomBall();
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery: Don’t create several instances of the same class if you can use only one instance for your purposes